Gopkg.toml

```diff
@@ -151,8 +152,9 @@
   revision = "36e9d2ebbde5e3f13ab2e25625fd453271d6522e"
 
 [[constraint]]
-  branch = "master"
+  # branch = "master"
   name = "github.com/status-im/rendezvous"
+  revision = "6aa6103545295820fb9a680b0da080e8cdf45def"
```

Comment:
you can pull changes to a branch with `dep ensure -update github.com/status-im/rendezvous`.
it will automatically update revision in Gopkg.lock